src/mjbots_robot_interface.cpp

```diff
@@ -80,6 +82,14 @@ MjbotsRobotInterface::MjbotsRobotInterface(const std::vector<JointMoteus> &joint
   moteus_options.imu_mounting_deg = imu_mounting_deg;
   moteus_interface_ = std::make_shared<::mjbots::moteus::Pi3HatMoteusInterface>(moteus_options);
 
+  // Initialize attitude shared pointer
+  attitude_ = std::make_shared<::kodlab::Attitude<float>>();
+  kodlab::rotations::EulerAngles<float> imu_world_offset =
```

Comment:
we probably want to make a free function that does this conversion. We can think about where to include it.

Reply:
Agreed, we need a header of Mjbots free functions somewhere in the `mjbots` namespace.  However, I think this may deserve a separate PR once this one goes through.

Reply:
Agreed, let's an issue to that end.

Reply:
#29